teams/teams-ps/teams/Set-CsTenantFederationConfiguration.md

```diff
@@ -439,6 +448,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ExternalAccessWithTrialTenants
+When set to 'Blocked', all external access with Teams subscriptions that are all trial will be blocked. Users from trial tenants will not be able to reach to users from tenants that have enabled the Block and vice-versa. The setting will also remove trial tenant users from existing chats if the block gets enabled.
```

Comment:
Should read:

When set to 'Blocked', all external access with users from Teams subscriptions that contain only trial licenses will be blocked. This means users from these trial-only tenants will not be able to reach to your users via chats, Teams calls, and meetings (using the users authenticated identity) and your users will not be able to reach users in these trial-only tenants. If this setting is set to "Blocked", users from the trial-only tenant will also be removed from existing chats.

Reply:
Updated.